### PR TITLE
f-form-field@4.7.1 - Visual Tests

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
v4.7.1
------------------------------
*February 7, 2022*

### Added
- `FormField.stories.js` with all variants of f-form-field on a single storybook page.
- `f-form-field--visual.component.js` and visual tests.

### CODE REVIEW

**Review by LTurns — DISMISSED** [review]

File: stories/standards/visual-regression-setup.stories.mdx [hunk 1]
```diff
@@ -118,6 +118,11 @@ This removes the risk of checking it into Git, and also prevents the need to mod
 set PERCY_TOKEN_F_YOUR_COMPONENT_NAME_HERE=<paste your token here>
 ```
 
+Once you have added this, to allow your PR to run the visual tests on CI, you'll also need to add this line below to your component's `package.json` prefacing 'YOUR_COMPONENT_TYPE_HERE' with either `atom`, `molecules`, `organisms` or `pages`.
```
Comment (xander-marjoram) [inline stories/standards/visual-regression-setup.stories.mdx]:
```suggestion
Once you have added this, to allow your PR to run the visual tests on CI, you'll also need to add this line below to your component's `package.json` replacing 'YOUR_COMPONENT_TYPE_HERE' with either `atoms`, `molecules`, `organisms` or `pages`.
```

File: stories/standards/visual-regression-setup.stories.mdx [hunk 2]
```diff
@@ -118,6 +118,11 @@ This removes the risk of checking it into Git, and also prevents the need to mod
 set PERCY_TOKEN_F_YOUR_COMPONENT_NAME_HERE=<paste your token here>
 ```
 
+Once you have added this, to allow your PR to run the visual tests on CI, you'll also need to add this line below to your component's `package.json` replacing 'YOUR_COMPONENT_TYPE_HERE' with either `atom`, `molecules`, `organisms` or `pages`.
```
Comment (xander-marjoram) [inline stories/standards/visual-regression-setup.stories.mdx]: Sorry think you missed this bit too
```suggestion
Once you have added this, to allow your PR to run the visual tests on CI, you'll also need to add this line below to your component's `package.json` replacing 'YOUR_COMPONENT_TYPE_HERE' with either `atoms`, `molecules`, `organisms` or `pages`.
```

**Review by xander-marjoram — DISMISSED** [review]

**Review by oliversweb — DISMISSED** [review]

**Review by oliversweb — DISMISSED** [review]

File: packages/components/atoms/f-form-field/test/visual/f-form-field.visual.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const FormField = require('../../test-utils/component-objects/f-form-field--visual.component');
+
+let formField;
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-form-field/test/visual/f-form-field.visual.spec.js]: It is better to scope this within the describe block. You could add it inside the first describe.

Comment (JoshuaNg2332) [reply packages/components/atoms/f-form-field/test/visual/f-form-field.visual.spec.js]: Ok. I'll add to that now.

**Review by kevinrodrigues — APPROVED** [review]

**Review by oliversweb — APPROVED** [review]

**Pull request merged**